docs/topics/native/native-ios-integration.md

```diff
@@ -1,8 +1,9 @@
-[//]: # (title: iOS integration)
+[//]: # (title: Integration with Swift/Objective-C ARC)
 
-Integration of Kotlin/Native garbage collector with Swift/Objective-C ARC is seamless and generally requires no additional
-work to be done. Learn more about [Swift/Objective-C interoperability](native-objc-interop.md).
+Kotlin and Objective-C use different memory management strategies, Kotlin has a tracing garbage collector,
+while Objective-C relies on automatic reference counting (ARC).
 
+The integration between them is usually seamless and generally requires no additional work.
```

Comment:
Being a new paragraph, it's worth to clarify what we refer with "them" and mention the nouns again.

Reply:
I think mentioning nouns would bring too much repetition, but I clarified the pronoun though